src/components/Layout/Section/RSVDetails.tsx

```diff
@@ -22,121 +22,133 @@ const stats = {
-          [8.369859, 49.39425],
-          [8.37038, 49.39447],
-          [8.371729, 49.39481],
-          [8.37255, 49.39492],
-          [8.37322, 49.39493],
-          [8.37345, 49.39494],
-          [8.37434, 49.3951],
-          [8.37442, 49.39515],
-          [8.374459, 49.39595],
-          [8.37451, 49.39607],
-          [8.37452, 49.39621],
-          [8.37451, 49.39635],
-          [8.37448, 49.39658],
-          [8.374459, 49.39705],
-          [8.37444, 49.39821],
-          [8.374449, 49.39837],
-          [8.37448, 49.39843],
-          [8.3744, 49.39852],
-          [8.37438, 49.39903],
-          [8.374359, 49.39954],
-          [8.37429, 49.40017],
-          [8.37407, 49.40076],
-          [8.373839, 49.40135],
-          [8.37334, 49.40235],
-          [8.37331, 49.40256],
-          [8.373419, 49.40276],
-          [8.37351, 49.40283],
-          [8.37373, 49.4031],
-          [8.37392, 49.40333],
-          [8.3742, 49.40347],
-          [8.3747, 49.40368],
-          [8.37484, 49.40377],
-          [8.37533, 49.40444],
-          [8.37601, 49.4055],
-          [8.376289, 49.40604],
-          [8.37646, 49.40633],
-        ],
-      },
-      properties: {},
-    },
-  ],
-};
+const mapBounds: LngLatBoundsLike = [
+  4.98865807458, 47.3024876979, 16.0169958839, 54.983104153,
+];
+function bounds(geometries) {
+  let [maxLat, maxLon] = [-Infinity, -Infinity];
+  let [minLat, minLon] = [Infinity, Infinity];
+  geometries.nodes.forEach(({ geometry: { coordinates } }) => {
+    coordinates.forEach((line) => {
+      const Xs = line.map((x) => x[1]);
+      const Ys = line.map((x) => x[0]);
+      maxLat = Math.max(...Xs, maxLat);
+      maxLon = Math.max(...Ys, maxLon);
+      minLat = Math.min(...Xs, minLat);
+      minLon = Math.min(...Ys, minLon);
+    });
+  });
+  return [minLat, minLon, maxLat, maxLon];
+}
+function center(geometries) {
+  const [minLat, minLon, maxLat, maxLon] = bounds(geometries);
+  return [minLon / 2 + maxLon / 2, minLat / 2 + maxLat / 2];
+}
 
-const layerStyle: LineLayer = {
-  id: 'rsv',
-  type: 'line',
-  paint: {
-    'line-width': 10,
-    'line-color': '#000000',
-  },
-};
-
-export const RSVDetails: React.VFC<Props> = (props) => {
-  const { radschnellweg } = props;
+export const RSVDetails: React.VFC<Props> = ({ meta, geometries }) => {
+  const [lon, lat] = center(geometries);
   return (
     <div className="relative bg-white">
-      <div className="h-56 bg-emerald-300 sm:h-72 lg:absolute lg:left-0 lg:h-full lg:w-1/2">
+      <div className="h-56 rounded-br-2xl bg-emerald-300 shadow-xl sm:h-72 lg:absolute lg:left-0 lg:h-full lg:w-1/2">
         <Map
           initialViewState={{
-            longitude: 8.3,
-            latitude: 49.4,
+            longitude: lon,
+            latitude: lat,
             zoom: 8,
           }}
           mapLib={maplibregl}
-          style={{ display: 'cover' }}
+          style={{ display: 'cover', borderRadius: '0 0 1rem' }}
```

Comment:
Könnte man hier TW classes übergeben? (Habe spontan nichts gefunden in den Docs.)

Reply:
Habe ich versucht. Konnte leider nichts finden. Irgendwas der gleichen ist auf jeden fall noch nötig um den `border-radius` nur für desktops zu aktivieren.

Reply:
Wie wäre es mit: `md:border-b-2xl` ? Und was ist `display: cover`? ^^